cherrysrv/client.go

```diff
@@ -182,7 +183,7 @@ func (clt *Client) UpdateInMain(format string, args ...interface{}) {
 
 	broadcast := func(key string, client *Client) bool {
 
-		if clt == client { // we don't want to send the message to us
+		if clt == client || !client.isLogged() { // we don't want to send the message to us or non-logged users
```

Comment:
Pretty sure we need to check if a client is logged in here.

Reply:
The plan is to have logged and non logged clients in main so they can receive general notifications, like server being shut down.

I'm open to review it once we have the server live and we see how it works.